docs/src/man/basics.md

```diff
@@ -0,0 +1,1388 @@
+  994 │   993     30  male        3  own      little           little          ⋯
+  995 │   994     50  male        2  own      NA               NA
+  996 │   995     31  female      1  own      little           NA
+  997 │   996     40  male        3  own      little           little
+  998 │   997     38  male        2  own      little           NA              ⋯
+  999 │   998     23  male        2  free     little           little
+ 1000 │   999     27  male        2  own      moderate         moderate
+                                                  4 columns and 985 rows omitted
+
+julia> german.Age # all get the vector stored in our DataFrame without copying it
+1000-element Vector{Int64}:
+ 67
+ 22
+ 49
+ 45
+ 53
+ 35
+ 53
+ 35
+ 61
+ 28
+  ⋮
+ 34
+ 23
+ 30
+ 50
+ 31
+ 40
+ 38
+ 23
+ 27
+
+julia> german."Sex" # the same using string indexing
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+
+julia> german[:, 3] # note that this creates a copy
+1000-element PooledArrays.PooledVector{String, UInt32, Vector{UInt32}}:
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ "male"
+ ⋮
+ "male"
+ "male"
+ "male"
+ "male"
+ "female"
+ "male"
+ "male"
+ "male"
+ "male"
+```
+
+To get one row as a `DataFrame`, we can index as follows:
+```jldoctest dataframe
+julia> german[2:2, :]
+1×10 DataFrame
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │     1     22  female      2  own      little           moderate         ⋯
+                                                               3 columns omitted
+
+julia> german[3, :] # this produces a DataFrameRow which is treated as 1-dimensional object similar to a NamedTuple
+DataFrameRow
+ Row │ id     Age    Sex     Job    Housing  Saving accounts  Checking account ⋯
+     │ Int64  Int64  String  Int64  String   String           String           ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   3 │     2     49  male        1  own      little           NA               ⋯
+                                                               3 columns omitted
+```
+
+We can get a single cell or element with the same syntax to get an element of an array:
+```jldoctest dataframe
+julia> german[4, 4]
+2
+```
+
+or to get a new `DataFrame` that is subset of rows and columns:
+```jldoctest dataframe
+julia> german[4:5, 4:5]
+2×2 DataFrame
+ Row │ Job    Housing
+     │ Int64  String
+─────┼────────────────
+   1 │     2  free
+   2 │     2  free
+```
+
+Assignment of a scalar to a data frame can be done in ranges using broadcasting:
```

Comment:
why do you give this example here? The section promised to show how to set a column in a data frame.

Reply:
My aim is to show if you want to set other values in the column(to provide some range) then you can do this using broadcasting.

Reply:
And it would be a good example for this section. However your example dose not show "set other values in the column" but how to set multiple columns at once (which is a different operation).

Please read carefully https://dataframes.juliadata.org/latest/lib/indexing/ as this will give you an understanding how examples of `getindex`/`setindex!`/`broadcast!` should be structured.